tsAssignment1.js

```diff
@@ -0,0 +1,57 @@
+function rectangleArea(height, width) {
```

Comment:
where is your typescript syntax? 
```suggestion
function rectangleArea(height: number, width: number): number {
  return height * width;
}
```